stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Logical: Int64 Logical Functions.
+ */
+
+/**
+ * Computes the bitwise logical "not" of the given value.
+ *
+ * @param value: The given value
+ * @returns The bitwise logical "not" of the given value.
+ * 
+ * @since v0.2.0
+ */
 @disableGC
-export let rec lnot = (x: Int64) => {
-  let xv = WasmI64.load(WasmI32.fromGrain(x), 8n)
+export let rec lnot = (value: Int64) => {
+  let xv = WasmI64.load(WasmI32.fromGrain(value), 8n)
   let ptr = newInt64(WasmI64.xor(xv, 0xffffffffffffffffN))
   let ret = WasmI32.toGrain(ptr): Int64
-  Memory.decRef(WasmI32.fromGrain(x))
+  Memory.decRef(WasmI32.fromGrain(value))
   Memory.decRef(WasmI32.fromGrain(lnot))
   ret
 }
 
-
+/**
+ * Computes the bitwise logical "and" on the given operands.
```

Comment:
```suggestion
 * Computes the bitwise logical AND on the given operands.
```
See above. My other comment is that we should put the operator syntax in parens and backticks like in Option/Result.